data/tekton-pipelines/pipelines-rbac.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-modify-data-object
+  annotations:
+    "kubevirt.io/deploy-namespace": "kubevirt-os-images"
+roleRef:
+  kind: ClusterRole
+  name: modify-data-object-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-create-vm
+  annotations:
+    "kubevirt.io/deploy-namespace": "kubevirt-os-images"
+roleRef:
+  kind: ClusterRole
+  name: create-vm-from-manifest-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-wait-for-vmi
+  annotations:
+    "kubevirt.io/deploy-namespace": "kubevirt-os-images"
+roleRef:
+  kind: ClusterRole
+  name: modify-data-object-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-cleanup-vm
+  annotations:
+    "kubevirt.io/deploy-namespace": "kubevirt-os-images"
+roleRef:
+  kind: ClusterRole
+  name: cleanup-vm-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-wait-for-vmi
+roleRef:
+  kind: ClusterRole
+  name: modify-data-object-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-modify-data-object
+roleRef:
+  kind: ClusterRole
+  name: modify-data-object-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-create-vm
+roleRef:
+  kind: ClusterRole
+  name: create-vm-from-manifest-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: windows-pipelines-cleanup-vm
+roleRef:
+  kind: ClusterRole
+  name: cleanup-vm-task
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Can this become an issue when `pipeline` SA is already there, i.e. will ssp-operator and tektoncd/operator fight for it?

Reply:
Currently the SA is deployed only to ^openshift|^kube namespace. Howewer in the future there might be a time, when this bug will be fixed and ssp and tekton operator will fight over this SA. I will add a check to ssp if the SA already exists